service/src/main/resources/application.yml

```diff
@@ -41,6 +41,10 @@ cbas:
   context:
     # Should match wds.instanceId and sam.workspaceId:
     workspaceId: "00000000-0000-0000-0000-000000000000"
+    # workspaceCreatedDate is supplied by Leo.
+    # When testing locally, manually fill in this value, e.g. "2024-02-16T14:13:02.500743Z".
+    # BackfillOriginalWorkspaceIdService will be skipped if the default value below is used.
+    workspaceCreatedDate: "0000-01-01T00:00:00.000000Z"
```

Comment:
I wasn't fully sure why this needed to be a magic value instead of left as null or empty?

Reply:
there may be a better way to do all of this, but the reason I'm not using an empty string is because parsing this to an OffsetDateTime will fail here: https://github.com/DataBiosphere/cbas/pull/255/files#diff-aa4c1c1284253e9e5472f8179c0118a9e3f28f26eb2afdbeacd3c2a00ab3fba3R19

I assume null would cause it to fail too, but didn't confirm that

Reply:
I'll leave this null, and override it in the integration test github action